protocol-impl/src/test/java/io/camunda/zeebe/protocol/impl/JsonSerializableToJsonTest.java

```diff
@@ -825,7 +826,7 @@ final var record = new DeploymentDistributionRecord();
           "messageId": "test-id",
           "name": "test-message",
           "deadline": 22,
-          "tenantId": "<default>"
+          "tenantId": "foo"
```

Comment:
🔧 We should really start to use a better example tenant in tests 😄 What about `Acme`?